src/main/java/com/sprint/mission/discodeit/service/jcf/JCFChannelService.java

```diff
@@ -0,0 +1,69 @@
+package com.sprint.mission.discodeit.service.jcf;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.service.ChannelService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class JCFChannelService implements ChannelService {
+
+    //final을 사용하면 해당  entity는 수정할 수 없다.
+    final List<Channel> channeldata;
```

Comment:
외부에서 데이터 접근을 막기 위해서 private 접근자를 꼭 붙이는게 좋을 것 같아요.  
```suggestion
    private final List<Channel> channeldata;
```